tests/unit_tests/client/address_book/impl.cc

```diff
@@ -45,6 +45,46 @@
 /// @class SubscriptionFixture
 struct SubscriptionFixture {
 
+  struct AddressBookEntry
+  {
+   public:
+    AddressBookEntry() : m_Host(""), m_Address() {}
+
+    explicit AddressBookEntry(const std::string& subscription_line)
+        : m_Host(""), m_Address()
+    {
+      try
+        {
+          std::size_t pos = subscription_line.find('=');
+          if (pos == std::string::npos)
+            throw std::runtime_error(
+                "AddressBookEntry: invalid subscription line");
+          m_Host = subscription_line.substr(0, pos++);
+          kovri::core::IdentityEx ident;
+          ident.FromBase64(subscription_line.substr(pos));
+          m_Address = ident.GetIdentHash();
+        }
+      catch (...)
+        {
+          kovri::core::Exception ex;
+          ex.Dispatch(__func__);
+          throw;
+        }
+    }
+
+    inline const std::string& host() const { return m_Host; }
+
+    inline const kovri::core::IdentHash& address() const { return m_Address; }
```

Comment:
>inline

Not necessary as this is implicitly inlined. `inline`s are also suggestions which are not always observed by the compiler and may go ignored. The compiler may also inline when not explicitly stated.

Reply:
I removed the `inline` suggestion, obvious now it doesn't help. Will look more into `inline` for future use.